api/src/modules/scenarios-planning-unit/entities/scenarios-planning-unit.geo.entity.ts

```diff
@@ -0,0 +1,72 @@
+import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+import { LockStatus } from '../lock-status.enum';
+
+export const scenariosPuDataEntityName = 'scenarios_pu_data';
+
+const toLockEnum: Record<1 | 2, LockStatus> = Object.freeze({
+  1: LockStatus.LockedIn,
+  2: LockStatus.LockedOut,
+});
+
+const fromLockEnum: Record<LockStatus, 1 | 2 | null> = Object.freeze({
+  [LockStatus.Unknown]: null,
+  [LockStatus.LockedIn]: 1,
+  [LockStatus.LockedOut]: 2,
+});
+
+@Entity(scenariosPuDataEntityName)
+export class ScenariosPlanningUnitGeoEntity {
+  @PrimaryGeneratedColumn('uuid')
+  id!: string;
+
+  /**
+   * missing FK
+   */
+  @Column({
+    type: 'uuid',
+    name: 'pu_geom_id',
+  })
+  puGeometryId!: string;
+
+  /**
+   * missing FK
+   */
+  @Column({
+    type: 'uuid',
+    name: 'scenario_id',
+  })
+  scenarioId!: string;
+
+  /**
+   * missing FK
+   */
+  @Column({
+    type: 'int',
+    name: 'puid',
+  })
+  projectId!: number;
```

Comment:
this column is only a numeric id for each PU (unique within each scenario, I think we would need a constraint for this). It's called `puid` in the db as it maps directly to the same-named variable in Marxan dat files: given this, I'd suggest to name the property itself `puid` too, or if wanting to be more explicit `planningUnitId`, though this may even cause more confusion (because we have PU ids as in UUIDs and then again an id of each `scenarios_pu_data` row.

Reply:
@hotzevzl how about `planningUnitMarxanId` in this case to better expose its nature? The reason I am afraid to leave "original" names within code is that their... meaning is unclear.

Reply:
@kgajowy I would go with your suggestion, thanks.
I am ok with using the original names too - that "only" forces each dev to memorize the 133 pages of the Marxan manual 😬 , but in general we don't need to save every character like in a 1980s MSDOS dat file, so descriptive names are ok too (and leaves us with the pleasure of dealing with one of the two complex problems in computer science, i.e. naming things 😅 )

Reply:
fixed in https://github.com/Vizzuality/marxan-cloud/pull/148/commits/6396c14d671e06fbc11259fc539499d536454a85